examples/dgmulti_1d/elixir_burgers_gauss_shock_capturing.jl

```diff
@@ -0,0 +1,68 @@
+using Trixi
+using OrdinaryDiffEq
+
+equations = InviscidBurgersEquation1D()
+
+###############################################################################
+#  setup the GSBP DG discretization that uses the Gauss operators from Chan et al.
+
+surface_flux = flux_lax_friedrichs
+volume_flux = flux_ec
+
+polydeg = 3
+basis = DGMultiBasis(Line(), polydeg, approximation_type = GaussSBP())
+
+indicator_function(u, ::InviscidBurgersEquation1D) = u[1]
+
+indicator_sc = IndicatorHennemannGassner(equations, basis,
+                                         alpha_max = 0.5,
+                                         alpha_min = 0.001,
+                                         alpha_smooth = true,
+                                         variable = indicator_function)
```

Comment:
We can also use the generic `first` function that we have specialized for these use cases:

```suggestion
indicator_sc = IndicatorHennemannGassner(equations, basis,
                                         alpha_max = 0.5,
                                         alpha_min = 0.001,
                                         alpha_smooth = true,
                                         variable = first)
```

Reply:
Good to know about `first`!